pkg/minikube/node/start.go

```diff
@@ -957,7 +957,7 @@ func addCoreDNSEntry(runner command.Runner, name, ip string, cc config.ClusterCo
 	}
 
 	if cc.DisableCoreDNSLog {
-		sed = fmt.Sprintf("%s -e '/^        log$/d'", sed)
+		sed = fmt.Sprintf("%s -e '/^        log *$/d'", sed)
```

Comment:
It will be easier to review if you squash this fix into the commit that added this change and force push.